data/src/main/java/io/gnosis/data/models/TransferDetails.kt

```diff
@@ -5,6 +5,8 @@ import io.gnosis.data.backend.dto.TxData
 import pm.gnosis.model.Solidity
```

Comment:
Why is this file called `TransferDetails`?

Reply:
Legacy. I renamed it